Gemfile

```diff
@@ -37,6 +37,7 @@ gem 'mongoid', '6.1.0'
 gem 'state_machines', '~> 0.4'
 gem 'state_machines-mongoid', '~> 0.1'
 gem 'statsd-ruby', '~> 1.4.0', require: false
+gem 'state_machines-graphviz'
```

Comment:
maybe this should be in a development group only, we don't really want/need it on production.

Reply:
moved